### DISCUSSION TITLE
排泄圈地。这也太胡扯了。庞氏骗局。信众多了好牟私利。智商筛选器

### DISCUSSION THREAD

**Discussion by Ocean9** [The-Run-Philosophy-Organization/run#513] [General]
挂羊头卖狗肉。根本不想让润人好过。
https://github.com/Ocean9/truth

**Comment by ghost** [comment]
第一，这个仓库怎么就恨国了？请指出此仓库涉嫌恨国的部分
而且 Github 可以自行 Fork 另起炉灶，不一定非要向上游提交 Pull requests
我个人不太喜欢 bboceng（@the-run-philosophy 这个帐号的所有者并不是 lidangzzz 而是 bboceng ），具体可见他们之间的推文（

第二，现在此仓库并没有包含飞跃手册的内容，此部分已经被移除

第三，关于官方的说法，似乎是 bboceng 自封的 ...
